PeerCo/small_scale_self_consumed_solar.md

```diff
@@ -1 +1,269 @@
+
+6. the asset owner or operator is not already being rewarded for self-consumption by another entity (e.g. government subsidies, ...)
+
+
+### Requirements on the system during the project lifetime
+
+During the whole duration of the project, the PV asset owner shall demonstrate proof of ownership of the asset and continued operation of the self-consumption. This documentation shall be made via all of the following:
+
+- sharing of consumption data throughout the period,
+
+- in the case of usage of the environmental attribute in reporting:
+    - contractual renunciation to the possibility of transfer of the environmental assets to a third party.
+
+- in the case of sales of the environmental attribute to a third party:
+    - contractual transfer of the environmental attribute ownership of the self-consumed generation,
+    - carbon accounting for the site (location and market-based) should reflect the sale of the carbon reduction corresponding to the transfer of the certificate.
+
+
+### Requirements on the operating context
+
+The project country shall meet the following criteria:
+1. delivery of micro-solar is not a requirement for building development or to meet an enforced carbon quota,
+2. emission intensity data is available at the hourly (or sub-hourly) level for the national or local grid.
+
+In cases where this methodology is used to generate carbon credits, extra requirements shall be met:
+
+3. PV arrays are not installed on more than 10% of all buildings in the market of operation
+
+
+## Project boundary
+
+GHG sources included or excluded from the project boundary.
+
+| Source                     | Gas     | Included | Justification           |
+| -------------------------- |---------|----------|------------------------ |
+| Grid electricity displaced | $CO_2$  | Yes      | Main emission source    |
+| Grid electricity displaced | $NO_x$, $SO_x$  | No       | Emissions data is not sufficiently granular and available the half hour or hour |
+
+
+## Measurement
+
+This section introduces the underlying data requirements.
+
+### Asset data
+
+PV production data shall be acquired via automatic digital communication to PeerCo's cloud once the project is established. However, in the initial phase of setting up a project (qualifying the site, its data and quantifying the benefits of the scheme), offline extractions of the data in an open format (which may be CSV, XLSX, etc.) may be accepted.
+
+All self-consumption data (and related generation data) shall be quality controlled prior its usage to generate verified certificates. This verification shall include:
+1. the maximum PV generation is in line with the PV asset's nominal rating,
+2. self-consumption is always lower than the total consumption,
+3. no significant PV generation is happening when there is no solar irradiance (e.g. at night).
+
+Where data is missing, filling of the gaps sall be allowed under the following conditions:
+1. data is not missing more than 5% of the time,
```

Comment:
Propose adding in a time period, i.e. over a trailing week or month period.

Reply:
Agreed. I'm adding a "per calendar month" in a new commit below.